Description: disable qtlocation for now, it is not yet part of Qt
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-needed
Last-Update: 2014-12-27

--- a/src/src.pro
+++ b/src/src.pro
@@ -14,22 +14,6 @@
 }
 
 qtHaveModule(quick) {
-    SUBDIRS += 3rdparty
-
-    location.depends = positioning 3rdparty
-    SUBDIRS += location
-
-    plugins.depends += location
-    imports.depends += location
-
-    contains(QT_CONFIG, private_tests) {
-        location_doc_snippets.subdir = location/doc/snippets
-        #plugin dependency required during static builds
-        location_doc_snippets.depends = location plugins
-
-        SUBDIRS += location_doc_snippets
-    }
-
     imports.depends += positioning
     SUBDIRS += imports
 }
